.github/workflows/other.yml

```diff
@@ -0,0 +1,25 @@
+name: Other
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: check ${{ matrix.tox_env }}
+    runs-on: Ubuntu-latest
+    strategy:
+      matrix:
+        tox_env:
+        - readme
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: setup Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: install tox
+      run: python -m pip install tox
+    - name: run check for ${{ matrix.tox_env }}
+      run: python -m tox -e ${{ matrix.tox_env }}
```

Comment:
I'm getting confused. Can we not have this as part of the "lint" checks, I feel it fits better in there.

I'd be OK with merging this and fixing that later, but it'll have to wait on the question about the setup.cfg formatter.

Also, I'm finding that the github UI is getting very confusing here. You seem to have pushed extra commits, but I only saw them by carefully checking what had been added. I don't know if that was related to the fact that you force-pushed, or if it's because you added new commits with extra changes in different files, but if you could do anything to simplify the review process, I'd appreciate it. (In particular, while you're still working on a PR, please try to avoid force-pushes and history edits - once the PR looks OK, if you want to tidy up the commit history then, let's do it as a single pass, although personally I'm fine with not doing that and just merging with the actual commit history as developed).

Reply:
> I'm getting confused. Can we not have this as part of the "lint" checks, I feel it fits better in there.

Well, this is a check on the binary packages generated (sdist and wheel) so they don't link check the existing code (that pre-commit does). So I don't think it can fall under that umbrella, and there's no way to force put it under AFAIK (unless we write a pre-commit plugin for it, but again as it's not a lint check I don't think it should be there).